docs/core-concepts/console/index.md

```diff
@@ -1,4 +1,7 @@
 # Writing console applications 
 
 *  [ Overview and moving pieces](overview.md)
+*  [ Console applications paradigm](paradigm.md)
+*  [ Creating single binaries with native compilation](single-binaries.md)
```

Comment:
This can be removed actually. We will not be shipping this capability any time soon.